Chatkit/Redux/State/RoomState.swift

```diff
@@ -0,0 +1,19 @@
+
+struct RoomState: State {
+    
+    // MARK: - Properties
+    
+    let identifier: String
+    let name: String
+    let isPrivate: Bool
+    let pushNotificationTitle: String?
```

Comment:
Are you missing `createdById`?

Reply:
I thought that we want to drop `createdBy` from our API. Is this still correct @mdpye?

Reply:
Yeah, let's try. I think it's listed in the discovery doc

Reply:
👍  I'll make a note to drop it form the Wire models also.